presidio-analyzer/presidio_analyzer/app.py

```diff
@@ -0,0 +1,196 @@
+# pylint: disable=wrong-import-position,wrong-import-order
+import grpc
+from concurrent import futures
+import os
+import sys
+import time
+import yaml
+from google.protobuf.json_format import MessageToJson
+from knack import CLI
+from knack.arguments import ArgumentsContext
+from knack.commands import CLICommandsLoader, CommandGroup
+from knack.help import CLIHelp
+from knack.help_files import helps
+
+from analyzer_engine import AnalyzerEngine  # noqa
+from recognizer_registry.recognizer_registry import RecognizerRegistry  # noqa
+from nlp_engine import NLP_ENGINES  # noqa
+from presidio_logger import PresidioLogger  # noqa
+from protobuf_models import analyze_pb2, analyze_pb2_grpc
+
+logger = PresidioLogger("presidio")
+
+WELCOME_MESSAGE = r"""
+
+ _______  _______  _______  _______ _________ ______  _________ _______
+(  ____ )(  ____ )(  ____ \(  ____ \\__   __/(  __  \ \__   __/(  ___  )
+| (    )|| (    )|| (    \/| (    \/   ) (   | (  \  )   ) (   | (   ) |
+| (____)|| (____)|| (__    | (_____    | |   | |   ) |   | |   | |   | |
+|  _____)|     __)|  __)   (_____  )   | |   | |   | |   | |   | |   | |
+| (      | (\ (   | (            ) |   | |   | |   ) |   | |   | |   | |
+| )      | ) \ \__| (____/\/\____) |___) (___| (__/  )___) (___| (___) |
+|/       |/   \__/(_______/\_______)\_______/(______/ \_______/(_______)
+
+"""
+
+CLI_NAME = "presidio-analyzer"
+
+helps[
+    "serve"
+] = """
+    short-summary: Create a GRPC server
+                   - presidio-analyzer serve --grpc-port 3000
+"""
+
+helps[
+    "analyze"
+] = """
+    short-summary: Analyze text for PII
+                   - presidio-analyzer analyze --text "John Smith drivers
+                   license is AC432223" --fields "PERSON" "US_DRIVER_LICENSE"
+"""
+
+
+class PresidioCLIHelp(CLIHelp):
+    def __init__(self, cli_ctx=None):
+        super(PresidioCLIHelp, self).__init__(
+            cli_ctx=cli_ctx, privacy_statement="", welcome_message=WELCOME_MESSAGE
+        )
+
+
+def serve_command_handler(
+    enable_trace_pii,
+    env_grpc_port=False,
+    grpc_port=3000,
+    nlp_conf_path="conf/default.yaml",
```

Comment:
Deciding on the model/framework to use should be better documented. Adding details here as docstring as well as on the development.md file on how to set up different spacy and stanza models would be really helpful.

Reply:
Yea, this is a great idea.  I'll work on that.

Reply:
Thanks!